docs_src/getting-started/random-generator-device.yaml

```diff
@@ -1,41 +1,34 @@
-name: "RandNum-Device"
-manufacturer: "Dell Technologies"
-model: "1"
-labels:
- - "random"
- - "test"
-description: "random number generator to simulate a device"
-
-deviceResources:
-    -   
-        name: "randomnumber"
-        description: "generated random number"
-        attributes:
-            { type: "random" }
-        properties:
-            value:
-                { type: "INT32", readWrite: "R", defaultValue: "0.00", minimum: "0.00", maximum: "100.00"  }
-            units:
-                { type: "String", readWrite: "R", defaultValue: "" }
-
-deviceCommands:
-    -
-        name: "Random"
-        get:
-            -
-                { operation: "get", object: "randomnumber", property: "value", parameter: "Random" }
-
-coreCommands:
-  -
-    name: "Random"
-    get:
-        path: "/api/v1/device/{deviceId}/Random"
-        responses:
-          -
-            code: "200"
-            description: ""
-            expectedValues: ["Random"]
-          -
-            code: "503"
-            description: "service unavailable"
+name: "RandNum-Device"
+manufacturer: "Dell Technologies"
+model: "1"
+labels:
+ - "random"
+ - "test"
+description: "random number generator to simulate a device"
+
+deviceResources:
+    -   
+        name: "randomnumber"
+        description: "generated random number"
+        attributes:
+            { type: "random" }
+        properties:
+            value:
+                { type: "int32", readWrite: "R" }
+            units:
+                { type: "String", readWrite: "R" }
+
+coreCommands:
+  -
+    name: "Random"

```

Comment:
this command can't match any DeviceCommand or DeviceResource.  It should be `randomnumber`